reconcile/gql_definitions/common/saas_files.py

```diff
@@ -84,7 +84,29 @@
     }
   }
   cluster {
-    ...OcConnectionCluster
```

Comment:
nit: Why not adjust the fragment with the additional parameters instead?

Reply:
I don't like to misuse fragments for other use cases. It looks like the `OcConnectionCluster` fragment defines just the needed attributes for `OcMap`.